app/workers/rate_limit_checker_worker.rb

```diff
@@ -1,5 +1,6 @@
-class RateLimitCheckerJob < ApplicationJob
-  queue_as :rate_limit_checker
+class RateLimitCheckerWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: :default, retry: 10
```

Comment:
This `:retry` option here tells sidekiq that we only want to retry the job 10 times. By default [Sidekiq will retry a job 25 times over the span of 21 days](https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry). Rather than letting jobs hang out in the retry queue that long lets fail a little faster, 10 tries allows the job to attempt to run for 4 hours before giving up.